index.html

```diff
@@ -0,0 +1,213 @@
+                </div>
+                <p class="program-p">BON JOVI in Live on Friday. Playing while 3 hours!</p>
+               </li>
+               <li class="grid-item">
+                <img class="program-img" alt="image-band" src="css/img/band-4.jpg">
+                <div class="cnt-title">                
+                    <h3 class="program-title">4NOS</h3>
+                </div>
+                <p class="program-p">CUARTETO DE NOS in Live on Saturday. Playing while 3 hours!</p>
+               </li>
+               <li class="grid-item">
+                <img class="program-img" alt="image-band" src="css/img/band-5.jpg">
+                <div class="cnt-title">                
+                    <h3 class="program-title">Milky</h3>
+                </div>
+                <p class="program-p">MILKY CHANCE in Live on Saturday. Playing while 2 hours!</p>
+               </li>
+            </ul>
+        </div>
+        <button id="join">Yesterland Tickets</button>
+    </section>
+    <section id="about" class="about">
+        <div id="about-container">
+            <h3 class="small-heading">Arrives in Urugay...</h3>
+            <h2 id="big-heading-about">   
+                Yesterland </br>
+                Music Festival 2024
+            </h2>
+            <div id="about-description">
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Maxime nulla alias quas facilis quibusdam nisi nostrum totam minus ipsa. Quae est iusto aliquam. Deserunt nesciunt cumque debitis, rerum maiores perferendis.Maxime nulla alias quas facilis quibusdam nisi nostrum totam minus ipsa. Quae est iusto aliquam. Deserunt nesciunt cumque debitis, rerum maiores perferendis.</p>
+            </div>
+            <p id="about-p">Please contact us per Email for any further questions about Yesterland Music Festival<p>
+            <h4 id="about-email">contact@yesterland.com</h4>
+        </div>
+    </section>
+    <section id="interviwes">
+        <h2 id="bands-h">Interviews</h2>
+        <hr class="hr orange">
+        <div id="bands-c">
+            <ul id="band-grid">
+                <li id="b-1" class="band-item">
+                    <div id="bg-perfil">
+                        <img class="bg-perfil" alt="" src="css/img/pngwing.com.png">
+                        <img class="perfil" alt="image singer" src="css/img/perfil-1.jpg">
+                        <img class="hidden" alt="" src="css/img/band-1-1.jpg">
+                    </div>
+                    <div class="content">
+                        <h3 class="name">Anthony Kiedis</h3>
+                        <p class="band-group">Red Hot Chili Pepers</p>
+                        <hr class="hr">
+                        <p class="i-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Praesentium culpa iusto nobis, soluta vero harum, dolor voluptates obcaecati non blanditiis odio?.</p>
+                    </div>
```

Comment:
- [ ] According to [the project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md), The section "Featured speakers" should be created dynamically in JavaScript. You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.